external/kafka-0-10-token-provider/src/test/scala/org/apache/spark/kafka010/KafkaDelegationTokenTest.scala

```diff
@@ -70,15 +71,19 @@ trait KafkaDelegationTokenTest extends BeforeAndAfterEach {
     Configuration.setConfiguration(new KafkaJaasConfiguration)
   }
 
-  protected def addTokenToUGI(): Unit = {
+  protected def addTokenToUGI(identifier: String, tokenService: Option[String] = None): Unit = {
```

Comment:
This signature is throwing me off when trying to understand the tests.

It seems that if you set the token service, then `identifier` is completely unused. Wouldn't it be simpler to just have the token service as an argument?

Reply:
Reading this part back, well... throwing me off as well so fixed.